src/adaptors/liqwid/index.js

```diff
@@ -46,23 +47,19 @@ const apy = async () => {
   const getPool = (market) => {
     return {
       pool: registryData.find(
-        (script) => script.name === `Liqwid${market.id}Action`
+        (script) => script.name === `Liqwid${market.id}BatchFinal`
```

Comment:
lets revert this change. otherwise i need to manually update pool values again in the db. so better to keep it as is